### PR TITLE
Adding Files for Chrome Extension

### PR DESCRIPTION
**Pull request by sagar7p**
Adding a basic chrome extension built with react and typescript

### CODE REVIEW

File: chrome-extension/dist/popup.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>Chrome Extension (built with TypeScript + React)</title>
```
Comment (saxena1) [inline chrome-extension/dist/popup.html:5]: Reeeeeeeesh

File: chrome-extension/src/eventPage.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// Listen to messages sent from other parts of the extension.
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+    // onMessage must return "true" if response is async.
+    let isResponseAsync = false;
```
Comment (vivekmogalapalli) [inline chrome-extension/src/eventPage.ts]: const?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,13 @@
-# trouver
-The ultimate planning and aggregation tool to find your next adventure
+# Trouver
+**The ultimate planning and aggregation tool to find your next adventure**
```
Comment (saxena1) [inline README.md:2]: We might want to use a less techy word. Though this can be addressed later

File: chrome-extension/package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "node-sass": "^4.13.1",
+    "sass-loader": "^6.0.7",
+    "style-loader": "^0.20.3",
+    "ts-loader": "^4.0.1",
+    "typescript": "^3.8.0",
+    "webpack": "^4.42.1",
+    "webpack-cli": "^3.3.11",
+    "webpack-merge": "^4.2.2"
+  },
+  "dependencies": {
```
Comment (vivekmogalapalli) [inline chrome-extension/package.json:26]: Might want to add prettier

File: chrome-extension/src/popup/Popup.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    align-items: center;
+    background-color: $background-color;
+    border: 2px solid $primary-color;
+    color: $primary-color;
+    display: flex;
+    font-size: 20px;
+    height: 75px;
+    justify-content: center;
+    width: 150px;
+}
```
Comment (vivekmogalapalli) [inline chrome-extension/src/popup/Popup.scss]: github's complaining on my end that there's no new line

File: chrome-extension/dist/popup.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    <script src="js/popup.js"></script>
+</head>
+
+<body>
+    <div id="popup"></div>
+</body>
+
+</html>
```
Comment (vivekmogalapalli) [inline chrome-extension/dist/popup.html]: github's complaining here about a missing newline.

File: chrome-extension/.gitignore [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+*.tgz
+my-app*
+template/src/__tests__/__snapshots__/
+lerna-debug.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+/.changelog
+.npm/
+yarn.lock
```
Comment (vivekmogalapalli) [inline chrome-extension/.gitignore]: missing newline here too :(

File: .gitignore [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+# OS generated files #
+######################
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
```
Comment (vivekmogalapalli) [inline .gitignore]: last place for the missing newline - commenting on all since I'm not sure if you see it too on your end :)

**Review by vivekmogalapalli — APPROVED** [review]

File: chrome-extension/package.json (hunk 4 above)
Comment (sagar7p) [reply chrome-extension/package.json:26]: added

**Pull request merged**